docs/integration/KafkaIntegration.md

```diff
@@ -8,104 +8,164 @@ sidebar_position: 1
 
 To better understand how Nussknacker works with Kafka, it's recommended to read the following first:
 * [Kafka introduction](https://kafka.apache.org/intro)
-* [Role of Schema Registry](/about/TypicalImplementation)
+* [Role of Schema Registry](/about/TypicalImplementationStreaming)
 * [Confluent Schema Registry](https://docs.confluent.io/platform/current/schema-registry/index.html)
 
 If you want to use Flink engine, this is also recommended:
 * [Flink keyed state](https://ci.apache.org/projects/flink/flink-docs-master/docs/concepts/stateful-stream-processing/#keyed-state)
 
 ## Concepts
 
+### Sources and sinks
+
 Kafka topics are native streaming data input to Nussknacker and the native output where results of Nussknacker scenarios processing are placed. 
-In Nussknacker terminology input topics are called *sources*, output topics are called *sinks*. This section provides important details of Nussknacker's integration with Kafka and Schema Registry.
+In Nussknacker terminology input topics are handled by *source* components, output topics are handled by *sink* components.
+This section provides important details of Nussknacker's integration with Kafka and Schema Registry.
 
-## Schema Registry integration
+### Schemas
 
-Nussknacker integrates with Schema Registries. It is the source of knowledge about:
-- Which topics are available in Kafka sources and sinks
-- What schema versions are available for topic
-- How code completions and validations on types described by schemas should work
-- How messages will be serialized and deserialized
+Nussknacker uses schemas for code completions and validations of messages. 
+Schema of message can be described in [Avro Schema format](https://avro.apache.org/docs/#schemas) or [JSON Schema format](https://json-schema.org) (Confluent Schema Registry only)
 
-Currently, Nussknacker supports two implementations of schema registries: based on *Confluent Schema Registry* and based on *Azure Schema Registry*. 
-During runtime, it serializes and deserializes messages in the way that is compatible with standard Kafka serializers and deserializers 
-delivered by those schema registries providers. Thanks to that you should be able to send messages to Nussknacker and read messages 
-produced by Nussknacker using standard tooling available around those Schema Registries.
+Schemas are retrieved from Schema Registry - *Confluent Schema Registry* and *Azure Schema Registry* are supported.
```

Comment:
are retrieved -> are stored 
                      -> are stored and retrieved

Reply:
I originally wrote sth like this, but after that I realize that it can be confusing, because someone can think that he/she can manage (add/remove) schemas in Nussknacker. But retrieved indeed looks "technical". Maybe Schemas "are managed" by Schema Registry?